src/test/java/com/microsoft/azure/kusto/kafka/connect/sink/E2ETest.java

```diff
@@ -31,20 +30,19 @@
     private static final String testPrefix = "tmpKafkaE2ETest";
     private String appId = System.getProperty("appId");
     private String appKey = System.getProperty("appKey");
-    private String authority = System.getProperty("authority", "microsoft.com");
+    private String authority = System.getProperty("authority");
     private String cluster = System.getProperty("cluster");
     private String database = System.getProperty("database");
     private String tableBaseName = System.getProperty("table", testPrefix + UUID.randomUUID().toString().replace('-', '_'));
     private String basePath = Paths.get("src/test/resources/", "testE2E").toString();
 
     @Test
-    @Ignore
```

Comment:
We havent configured the e2e environment yet for this repo . This will fail the tests